features/product/managing_products/modifying_taxons_assigned_to_an_existing_product.feature

```diff
@@ -0,0 +1,18 @@
+@managing_products
```

Comment:
Add also scenarios for adding the new taxon to the product and removing one.

Reply:
I added test for creation but I didn't add test for removing because I didn't add this endpoint. I only added POST and PUT endpoint and both are tested